doc/md_src_files/060_decision-trees.md

```diff
@@ -128,7 +128,7 @@ Coordinators are particularly interested in facilitating communication because t
 This section handles three aspects of this challenge: formats for communicating SSVC, how to handle partial or incomplete information, and how to handle information that may change over time.
 
 This section is about communicating SSVC information about a specific vulnerability.
-A supplier making a decision on allocating effort or a deployer should have a decision tree and it's decision points and possible values specified already.
+A supplier deciding on allocating effort or a deployer should have a decision tree and its decision points and possible values specified already.
```

Comment:
Suggested rewrite to group the subject nouns and the adjective clause (which I think applies to both nouns in the subject):
"Both suppliers and deployers, when deciding to allocate effort, should have a decision tree and its decision points and possible values specified already."

Reply:
"Both suppliers and deployers, when deciding to allocate effort, should maintain a decision tree and its decision points and possible values (?)already defined(?)."

specified already
defined already

Reply:
"should have a decision tree and its decision points and possible values specified already" --> "should already have a decision tree with some of its decision points and possible values defined"?

I'm not really sure what the point is that's being made with this sentence (due to my lack of cached context, that's not a critique of the author), I'm mostly just parsing the grammar. So it's possible that there's just another way to make the point that deconvolves the sentence structure by just replacing it with something simpler.

Reply:
I'm in the same position. (shrug)  
My interpretation of the original sentence: ... or a deployer has a decision tree and decision points and values already defined.

Reply:
How about?
> A stakeholder should have a decision tree, its decision points, and its prioritization labels defined in advance of any decision on allocating effort for or communicating about a particular vulnerability. 

Since this is in `## Guidance on Communicating Results` I think that ties it back together.